spec/valkyrie/persistence/solr/mapper_spec.rb

```diff
@@ -7,9 +7,12 @@
     instance_double(Book,
                     id: "1",
                     created_at: Time.zone.at(0),
-                    title: ["Test", RDF::Literal.new("French", language: :fr)],
-                    author: ["Author"],
-                    attributes: { title: nil, author: nil, id: nil })
+                    attributes:
+                    {
+                      created_at: Time.zone.at(0),
```

Comment:
The tests fail without it, but I can fix up `#created_at` to delegate to `#attributes`.